uber/config.py

```diff
@@ -412,3 +412,9 @@ def _is_intstr(s):
     c.VOLUNTEER_CHECKLIST = [url for step, url in _items]
 
 stripe.api_key = c.STRIPE_SECRET_KEY
+
```

Comment:
This is a slightly dangerous modification because there are actually three different things which in theory should be modified:
- ``c.DOOR_PAYMENT_METHOD_OPTS`` which is a ``list`` of ``tuple``s of the values/descriptions
- ``c.DOOR_PAYMENT_METHODS`` which is a dictionary with the same key/value pairs as the above list
- ``c.DOOR_PAYMENT_VARS`` which is a list of strings, where those strings are the attribute names on ``c`` for the constants defined by this enum, e.g. this would have ``['STRIPE', 'SQUARE', 'GROUP', ...]``

In this particular case we're probably only using ``c.DOOR_PAYMENT_METHOD_OPTS`` but I think this speaks to a need for a method on ``c`` which supports dynamically adding or removing options from these enums so that it can handle all of this transparently without us needing to remember or even know that this is going on behind the scenes.